samples/durable-task-sdks/dotnet/Agents/PromptChaining/AgentChainingSample.csproj

```diff
@@ -0,0 +1,31 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <!-- Generate assembly info from these properties -->
+    <GenerateAssemblyInfo>true</GenerateAssemblyInfo>
+    <Version>1.0.0</Version>
+    <AssemblyTitle>AgentChainingSample</AssemblyTitle>
+    <Description>Sample demonstrating agent prompt chaining with Durable Task</Description>
+    <Copyright>Copyright © 2025</Copyright>
+    <Company>Microsoft</Company>
+    <Product>AgentChainingSample</Product>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Azure.AI.Projects" Version="1.0.0-beta.9" />
+    <PackageReference Include="Azure.AI.Agents.Persistent" Version="1.0.0" />
+    <PackageReference Include="Azure.Identity" Version="1.13.1" />
+    <PackageReference Include="Microsoft.DurableTask.Client" Version="1.5.0" />
+    <PackageReference Include="Microsoft.DurableTask.Client.AzureManaged" Version="1.5.0-preview.1" />
+    <PackageReference Include="Microsoft.DurableTask.Worker" Version="1.5.0" />
+    <PackageReference Include="Microsoft.DurableTask.Worker.AzureManaged" Version="1.5.0-preview.1" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Logging.Console" Version="8.0.0" />
```

Comment:
We should consider using [central package management](https://learn.microsoft.com/nuget/consume-packages/central-package-management) for this repo. That will make it significantly easier to update dependencies over time.